### PR TITLE
chaw's first Goodie, spreading Happy Hacking wishes.

### PR DESCRIPTION
**Pull request by chaw**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**

Wishes everyone "Happy Hacking!"

**What problem does your Instant Answer solve (Why is it better than organic links)?**

It's a simple getting started exercise.

**What is the data source for your Instant Answer? (Provide a link if possible)**

None needed.

**Why did you choose this data source?**

NA.

**Are there any other alternative (better) data sources?**

NA.

**What are some example queries that trigger this Instant Answer?**

duckduckhack chaw

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

developers and techy folks.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

Only the tutorial.

**Which existing Instant Answers will this one supercede/overlap with?**

None.

**Are you having any problems? Do you need our help with anything?**

No.

**Where did you hear about DuckDuckHack? (For first time contributors)**

Colleague.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
duckduckhack101

## Checklist
Please place an 'X' where appropriate.

```
[X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility


    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera

Tested on Debian wheezy iceweasel.
```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/chaw.pm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/chaw.pm";
+attribution github => ["https://github.com/chaw/", "chaw"];
+
+# Triggers
+triggers start => "duckduckhack chaw";
+
+# Handle statement
+handle remainder => sub {
+	return if $_;
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/chaw.pm]: Can you fix this hard-tab? It should be 4 spaces :+1:

**Comment by moollaza** [comment]
LGTM :+1:

**Comment by moollaza** [comment]
@chaw thanks a lot for contributing and congrats on your first Goodie :cake:

**Comment by moollaza** [comment]
@chaw congrats! It's live: https://duckduckgo.com/?q=duckduckhack+chaw&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**